core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -233,8 +234,14 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
         kafkaScheduler.startup()
 
         /* create and configure metrics */
+        kafkaYammerMetrics = KafkaYammerMetrics.INSTANCE
+        kafkaYammerMetrics.configure(config.originals)
+
+        val jmxReporter = new JmxReporter(jmxPrefix, "kafka.")
```

Comment:
We don't generally prefix the broker's configuration keys with any special value.  This isn't really consistent with the rest of the broker config, so I would suggest removing it.

Reply:
see my comment above, I am only following conventions for existing JMX metrics configs. https://github.com/apache/kafka/blob/1b08fceb7a3c502d2d18dc99796101ed0b4ff8b7/core/src/main/scala/kafka/server/KafkaConfig.scala#L435
If this is not correct, we need to amend the KIP

Reply:
Let's continue the discussion in the other thread.